### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by varun-Tad**
Created login and sign up.Please review

### CODE REVIEW

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Give signup and login page padding or margin from bottom

File: Pages/Login-page/login.css [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+    --color-tertiary-opacity: #ff58602d;
+    --gradient-primary: linear-gradient(to top left, #39b385, #9be15d);
+    --gradient-secondary: linear-gradient(to top left, #ffb003, #ffcb03);
+  }
+
+
+*{
+    margin: 0;
+    box-sizing: border-box;
+
```
Comment (sri-rishi) [inline Pages/Login-page/login.css]: remove extra spacing

File: Pages/Login-page/login.css [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+body{
+    background-color: #f3f3f3;
+    overflow-y: hidden;
+}
+
+.main-navbar{
+    display:flex;
+    background-color: rgba(255, 255, 255, 0.95);
+    height:12rem;
+    display: flex;
```
Comment (sri-rishi) [inline Pages/Login-page/login.css]: you have used two time display: flex

File: Pages/Login-page/login.css [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+    align-items: center;
+    justify-content:center;
+    height: 5.5rem;
+    width: 5.5rem;
+    border-radius: 50%;
+    margin-bottom: 2rem;
+    padding-left:2rem;
+    padding-right:0.5rem;
+    margin-right:3rem;
```
Comment (sri-rishi) [inline Pages/Login-page/login.css]: You can give padding and margin as shorthand property

File: Pages/Login-page/login.css [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+    margin-top:1rem;
+   
+}
+/* MAIN CONTAINER */
+
+
+.main-container{
+    margin-top:3rem;
+    /* display: grid;
+    grid-template-columns: repeat(2,1fr); */
```
Comment (sri-rishi) [inline Pages/Login-page/login.css]: remove comments

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Comment by varun-Tad** [comment]
Thanks for the review @sri-rishi . Will make the changes

File: Pages/Login-page/login.css [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+} 
+
+.badge-portion{
+    display: flex;
+    align-items: center;
+    justify-content:center;
+    height: 5.5rem;
+    width: 5.5rem;
+    border-radius: 50%;
+    padding:0 0.5rem 0 2rem;
```
Comment (gauravsinhaweb) [inline Pages/Login-page/login.css]: use px for value less than 10px

File: Pages/Login-page/login.css [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+    width:40rem;
+    height:4rem;
+    background-color: var(--color-primary-opacity);
+    margin-right: 3rem;
+    margin-top:1rem;
+   
+}
+/* MAIN CONTAINER */
+
+
```
Comment (gauravsinhaweb) [inline Pages/Login-page/login.css]: remove extra spacing

File: Pages/Login-page/login.css [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+    height:200px;
+    
+}
+
+.image-container p{
+    position: relative;
+    top:24rem;
+    left:45rem;
+    font-size: 3rem;
+    color:#333;
```
Comment (gauravsinhaweb) [inline Pages/Login-page/login.css:122]: use variable colors

File: Pages/Login-page/login.css [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+    position: relative;
+    top:350px;
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px, rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px, rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+    transform: rotateZ(-10deg);
+}
+
+.image3{
+    position: relative;
+     left:300px;
+     bottom: 130px;
```
Comment (gauravsinhaweb) [inline Pages/Login-page/login.css]: use rems for responsiveness.

File: Pages/Login-page/login.html [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+<div class ="main-container">
+    <!-- <div class="image-container">
+        <img class="image1" src="../../images/watch.jpeg" alt="">
+        <img class="image2" src="../../images/sport-shoes.webp" alt="">
+        <img class="image3" src="../../images/Belt.webp" alt="">
+        <img class="image4" src="../../images/Bag.webp" alt="">
+        <p>...and more!</p>
+    </div>  -->
+
```
Comment (gauravsinhaweb) [inline Pages/Login-page/login.html]: remove unnecessary comments. Nice work!

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

**Comment by varun-Tad** [comment]
Will make the required changes.Thanks for the review @gauravsinhaweb

**Pull request merged**